bodhi/server/models/models.py

```diff
@@ -854,6 +854,7 @@ def edit(cls, request, data):
             comment += '\n\nRemoved build(s):\n'
             for removed_build in removed_builds:
                 comment += "\n- %s" % removed_build
+            comment += '\n\n\nKarma has been reset.'
```

Comment:
I think we still want to tell the user that the karma has been reset on new_builds too, for consistency.